src/components/ProjectName/ProjectName.jsx

```diff
@@ -74,30 +74,21 @@ const ProjectName = ({
     }
   };
 
+  const handleOnBlur = (event) => {
+    // If the blur event is triggered by clicking the tick button we want to make sure the target is clicked
+    if (event.relatedTarget?.className?.includes("project-name__button")) {
+      const target = event.relatedTarget ? event.relatedTarget : null;
+      target.click();
+    }
+    resetName(event);
+  };
```

Comment:
Can we not just not reset the name if the click is over the button and let the button handle the click event? I think we're trying to avoid having fake clicks firing off and things 🤔